app/models/commits.rb

```diff
@@ -0,0 +1,17 @@
+class Commits
```

Comment:
when creating a new instance of this class, it will represent a single entity, this should be singular 👍